xkit.js

```diff
@@ -3626,7 +3653,21 @@ function show_error_script(message) {
 
 function show_error_reset(message) {
 	// Shortcut to call when there is a javascript error.
-	XKit.window.show("XKit ran into an error.", "<b>Generated Error message:</b><br/><p>" + message + "</p>It is recommended that you reset XKit. Alternatively, you can write down the error message above and contact New XKit Support to see how you can fix this or reload the page to try again.", "error", "<div id=\"xkit-close-message\" class=\"xkit-button default\">OK</div><a href=\"http://www.tumblr.com/xkit_reset\" class=\"xkit-button\">Reset XKit</a><a href=\"https://new-xkit-support.tumblr.com\" class=\"xkit-button\">New XKit Support</a>");
+	XKit.window.show(
+		"XKit ran into an error.",
+
+		"<b>Generated Error message:</b><br/><p>" + message + "</p>" +
+		"It is recommended that you force-update XKit. " +
+		"Alternatively, you can write down the error message above and contact New XKit Support to see how you can fix this or reload the page to try again.",
```

Comment:
This sentence is a run-on, and it's not particularly helpful. It should be more proscriptive:

 - "If you've seen this message more then once, try restarting your browser. If you continue to see this message after completing the update process and restarting your browser, please contact us: [support discord link]"